src/introduction.md

```diff
@@ -1,46 +1,98 @@
 # Introduction
 
-The goal of this book is to provide a comprehensive guide on using the [Rust programming language] with [Espressif] SoCs and modules.
+The goal of this book is to provide a comprehensive guide on using the [Rust Programming Language] with [Espressif] devices.
 
-Rust support for these devices is still a work in progress, and progress is being made rapidly. Because of this, parts of this documentation may be out of date or change dramatically between readings.
+[rust programming language]: https://www.rust-lang.org/
+[espressif]: https://espressif.com/
 
-For tools and libraries relating to Rust on ESP, please see the [esp-rs organization] on GitHub.
+Rust support for these devices is still a work in progress, and progress is being made rapidly. Because of this, parts of this documentation may be out of date or change dramatically between readings.
 
-> #### A note in SoCs support.
->
-> The content of the books applies to ESP32, ESP32-S, and ESP32-C series of SoCs.
-> ESP8266 series is out of the scope of this book, support for Rust in ESP8266 is limited and only some parts
-> of the book are applicable for this SoC.
+For tools and libraries relating to Rust on ESP, please see the [esp-rs organization] on GitHub. This organization is managed by employees of Espressif as well as members of the community.
 
-[rust programming language]: https://www.rust-lang.org/
-[Espressif]: https://espressif.com/
 [esp-rs organization]: https://github.com/esp-rs/
 
-## Status of This Book
+> **A Note on Device Support**
+>
+> The contents of this book apply to the ESP32 series of devices only; this
+> includes:
+>
+> - ESP32 Series
+> - ESP32 C-Series
+> - ESP32 H-Series
+> - ESP32 P-Series
```

Comment:
Shall we include series that are not yet publicly available?

Reply:
They've been announced by Espressif and we will be supporting them (I have already started on H2 support) so I decided to include them. I guess we could remove them and add them back in later, though.